QA_TOKEN_SWAP.md

```diff
@@ -0,0 +1,52 @@
+# Token Swap
+
+1. **Token swap screen**
+	1. [ ] Open the token swap screen via the bottom menu
+    1. If the :arrows_clockwise: icon is not present then enable the feature flag `token-swap.rollout` for your device.
+	1. [ ] Check that HTR is already selected as the top token and the bottom token is already selected.
+  1. [ ] Check that both amounts should be set to 0.00
+
+1. **Token Selection**
+  1. [ ] Click on either token box to select another token
+    1. A list of tokens and balances should be shown
+  1. [ ] Close this screen and send some HTR to your wallet
+  1. [ ] Check that the HTR balance shows correctly on this screen
+
+1. **Token swap quote**
+  1. [ ] Clicking on the :arrows_clockwise: icon between the tokens should switch the top and bottom tokens.
+  1. [ ] Add 1.00 HTR on the top input and click outside the input area.
+    1. The bottom value should be filled out and in gray tone.
+  1. [ ] Check the swap quote information
+    1. Conversion rate should match the input/output amounts
+    1. Slippage should be 0.5%
+    1. Price impact should be a percentage
+    1. Minimum received should be a value lower than the output (bottom) amount
+  1. [ ] This time add 1.00 hUSDC on the bottom input area
+    1. The top value should be filled out and in gray tone.
+  1. [ ] Check the swap quote information
+    1. Conversion rate should match the input/output amounts
+    1. Slippage should be 0.5%
+    1. Price impact should be a percentage
+    1. Maximum to deposit should be a value higher than the input (top) amount
```

Comment:
chore: The formatting gets a bit off here. See below:

<img width="789" height="149" alt="Image" src="https://github.com/user-attachments/assets/423cbb55-0495-4641-bf2d-fd5638e0bb98" />

Maybe it's because the `[ ]` syntax does not allow for child elements, or maybe it's the amount of empty spaces not being recognized as a tab.

Reply:
Fixed, it does work but only with 4 spaces indentation.